Gemfile.lock

```diff
@@ -34,43 +34,43 @@ GEM
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.8.2)
+    i18n (1.8.3)
       concurrent-ruby (~> 1.0)
     json (2.3.0)
-    jwt (2.1.0)
-    jwt_signed_request (2.5.1)
-      jwt (>= 1.5.0, < 2.2.0)
+    jwt (2.2.1)
```

Comment:
### [jwt](https://github.com/jwt/ruby-jwt)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.1.0 → 2.2.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **83** commits. See the full changes on [the compare page](https://github.com/jwt/ruby-jwt/compare/v2.1.0...v2.2.1).

These are the first 10 commits:
- (19ca722) [Fix link format](https://github.com/jwt/ruby-jwt/commit/19ca722e1dc53e763e8ea503af011185077da50d)
- (949db0a) [Merge pull request #248 from y-yagi/patch-2](https://github.com/jwt/ruby-jwt/commit/949db0af05dcab4f25bb46be1c5fa19a03eb803c)
- (c910a40) [Fix README](https://github.com/jwt/ruby-jwt/commit/c910a405d8cf19aa7c21be30696009ab304970a4)
- (64466b5) [[CI] Test against Ruby 2.5](https://github.com/jwt/ruby-jwt/commit/64466b55d89471bb99acda1ae4ad98ad0a118fe8)
- (c0b4c59) [use first-class object short hand](https://github.com/jwt/ruby-jwt/commit/c0b4c59cdbe7fd0ca0646e5a80b3a8e8e128a4cd)
- (61b80b5) [Merge pull request #255 from akabiru/ch-error-class-shorthan…](https://github.com/jwt/ruby-jwt/commit/61b80b5f536d23ae20470b88af5c8645682c1800)
- (f82ef04) [Merge pull request #253 from nicolasleger/patch-1](https://github.com/jwt/ruby-jwt/commit/f82ef0499fd424765d3026e9770db3c215bfe095)
- (c6643e3) [Merge pull request #250 from rono23/fix-readme](https://github.com/jwt/ruby-jwt/commit/c6643e3cd4db378cada1739b9cfb6b4ec10151ad)
- (8384f0c) [removed leeway from verify_iat and tests](https://github.com/jwt/ruby-jwt/commit/8384f0cbd163a094a39b25c6f1eaaa625e08c9b8)
- (22b24f0) [added test for failing iat even with leeway](https://github.com/jwt/ruby-jwt/commit/22b24f0d9dc302443a116c4405d842e34a59f603)


</details>